lib/bolt/module_installer.rb

```diff
@@ -194,5 +194,29 @@ def install_puppetfile(path, moduledir, config = {})
 
       ok
     end
+
+    # Asserts that an existing Puppetfile is managed by Bolt.
+    #
+    private def assert_managed_puppetfile(puppetfile, path)
```

Comment:
I don't see where this gets called?

Reply:
Leftover from before the refactor of this class.